src/analysis/syntax.ts

```diff
@@ -1,4 +1,34 @@
-import * as SetOps from "../infra/setops";
+import * as SetOps from "infra/setops";
+
+export type TypeScriptSyntaxToken =
```

Comment:
these were actually on purpose in `syntax.d.ts` (which is a typescript definitions file) - any reason you reverted that change?

Reply:
Because I set `"isolatedModules": true` in `tsconfig.json` and `syntax.d.ts` is global. See [this post](https://stackoverflow.com/questions/42233987/how-to-configure-custom-global-interfaces-d-ts-files-for-typescript) for why I made this change. You can change it back later by adapting `tsconfig.json`, but I am not sure writing definition files in such a way is recommended, since we are not writing a library and the type files are not grouped under a folder.